pulp_2to3_migration/app/plugin/content.py

```diff
@@ -372,7 +373,8 @@ def get_remote_by_importer_id(importer_id):
                 pb.increment()
 
             if has_future:
```

Comment:
if has_future and dc:

Reply:
that was my first thought but then if has_future=True but there is no dc, a batch of futures might not be resolved.